Dockerfile.arm

```diff
@@ -24,10 +24,11 @@ RUN dotnet publish Jellyfin.Server --configuration Release --output="/jellyfin"
 
 
 FROM multiarch/qemu-user-static:x86_64-arm as qemu
-FROM debian:stretch-slim-arm32v7
+FROM debian:stretch-slim
```

Comment:
how would it know we want to target `arm32`?

Reply:
I think we might just be able to delete the second ARM file.

Reply:
I actually fail to see where it would know we want any `arm`...

Reply:
lsio images still cover 3 dockerfiles, one for each arch so i don't think we can delete the 64 version

Reply:
> how would it know we want to target `arm32`?

https://github.com/jellyfin/jellyfin/blob/3b3718e43749871c6c7e79bb62ffa1a382100728/Dockerfile.arm#L26

Reply:
If both build fine I'd say we can merge this and see what happens.

Reply:
RPi4 is aarch64 so I should be able to build it and test it when I get time